Go/port-scanner-go/readme.md

```diff
@@ -0,0 +1,29 @@
+
+# Port scanner in Go
+Go-lang script for scanning specific or all open ports of the device.
+
+# Packages Used
+1. fmt
+2. time
+3. sync
+4. strings
+5. net
+
+# How to run
+
+** Navigate to the project directory.
+
+```bash
+cd Rotten-Scripts/Go/port-scanner-go
+```
+```bash
+go run port_scanner.go 
+```
+## Output
+
+![](https://i.imgur.com/xMil1Kh.jpg)
+![](https://i.imgur.com/VEQPbz3.jpg)
+![](https://i.imgur.com/tMiEYNk.jpg)
+
+## Author
+  Pritam Pawar.
```

Comment:
Add a hyperlink to your LinkedIn/Github

Reply:
@HarshCasper I have made changes that you suggested